app/boards/arm/nrfmicro/pinmux.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2017 I-SENSE group of ICCS
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/gpio.h>
+#include <sys/sys_io.h>
+#include <devicetree.h>
+#include <pinmux/stm32/pinmux_stm32.h>
+
+static int pinmux_stm32_init(struct device *port)
```

Comment:
Fix the naming here, this should be `pinmux_nrfmicro_init` and at the end of the file.